the builds are failing (expected test failure?)

```
19:12:23 ======================================================================
19:12:23 ERROR: test_nested_inplace (__main__.TestJit)
19:12:23 ----------------------------------------------------------------------
19:12:23 Traceback (most recent call last):
19:12:23   File "test_jit.py", line 1171, in test_nested_inplace
19:12:23     self.assertExpectedTrace(trace)
19:12:23   File "test_jit.py", line 70, in assertExpectedTrace
19:12:23     self.assertExpected(str(trace), *args, **kwargs)
19:12:23   File "/var/lib/jenkins/workspace/test/common.py", line 362, in assertExpected
19:12:23     "python {} {} --accept").format(munged_id, s, __main__.__file__, munged_id))
19:12:23 RuntimeError: I got this output for TestJit.test_nested_inplace:
19:12:23 
19:12:23 graph(%0 : Double(2, 2)) {
19:12:23   %1 : Double(2, 2) = threshold[threshold={0}, value={0}](%0)
19:12:23   return (%1);
19:12:23 }
19:12:23 
19:12:23 
19:12:23 No expect file exists; to accept the current output, run:
19:12:23 python test_jit.py TestJit.test_nested_inplace --accept
```

--- a/test/expect/TestJit.test_nested_inplace.expect
+++ b/test/expect/TestJit.test_nested_inplace.expect
@@ -1,0 +1,4 @@
+graph(%0 : Double(2, 2)) {
+  %1 : Double(2, 2) = threshold[threshold={0}, value={0}](%0)
+  return (%1);
+}

